src/usr/local/www/head.inc

```diff
@@ -497,7 +498,7 @@ if (are_notices_pending()) {
 				?>
 					<li class="dropdown">
 						<a href="/index.php?logout">
-							<i class="fa fa-sign-out" title="<?=gettext("Log out")?>"></i>
+							<i class="fa fa-sign-out" title="<?=gettext("Logout") . " (" . $_SESSION['Username'] . "@" . htmlspecialchars($system_url) . ")"?>"></i>
```

Comment:
I was thinking more like `$_SERVER['REMOTE_ADDR']` to show where the user was logged in from (that's what we record in the config when they make changes)